app/views/admin/users/_user_results.html.haml

```diff
@@ -23,4 +25,9 @@
   = paginate @users
 
 - else
-  %p= "No users matched that search"
+  - if params[:search].present?
+    %p No users matched your search.
+  - else
+    %p
+      There are no users,
+      = link_to "click here to create a new user.", new_path_for_user
```

Comment:
https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

Don't use **click**
>Instead of using the word “click,” you might look for a different verb that relates to the user’s task. There’s always a better and more relevant verb to be used. “Click” makes users think of their mouse; but a task-related verb would make them think of the task itself and would keep them engaged with the content and focused on using the interface, not their mouse.

Don't use **here**
>Using the word “here” to make a link noticeable is unnecessary because that’s what the distinct styling of a link is supposed to do.

**Provide information** what the user can expect when clicking a link
>It’s also a good idea to choose the text of your link anchors as specific as possible. For example, if you’re linking to an article or book, it might be a good idea not to use the word “article” or “book” for the anchor. Instead, we could use the relevant title. This will give the user more detailed information about what they’re clicking to and what to expect.